io.github.insilmaril.vym.yml

```diff
@@ -0,0 +1,28 @@
+app-id: io.github.insilmaril.vym
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: vym
+rename-desktop-file: vym.desktop
+rename-icon: vym #128x128
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --own-name=org.insilmaril.*
```

Comment:
this should be changed upstream to use the same as the app-id. Registration of dbus name have similar constraint.